project/plugins.sbt

```diff
@@ -16,4 +16,6 @@ addSbtPlugin("com.eed3si9n" % "sbt-buildinfo" % "0.6.1")
 
 addSbtPlugin("com.eed3si9n" % "sbt-doge" % "0.1.5")
 
+addSbtPlugin("com.typesafe.sbteclipse" % "sbteclipse-plugin" % "5.1.0")
```

Comment:
Can we remove this? You can have it in a scalaide.sbt file that is ignored in you `~/.gitignore`.